Cairo Desktop/Cairo Desktop/CairoSettingsWindow.xaml.cs

```diff
@@ -511,5 +519,264 @@ public static CairoSettingsWindow Instance
                 return _instance;
             }
         }
+
+        #region Desktop Background
+        private void loadDesktopBackgroundSettings()
+        {
+            // Load only relevent Background Types
+            ComboBoxItem windowsDefaultBackgroundItem = new ComboBoxItem()
+            {
+                Name = "windowsDefaultBackground",
+                Content = Localization.DisplayString.sSettings_Desktop_BackgroundType_windowsDefaultBackground,
+                Tag = windowsImageBackgroundStackPanel
+            };
+            ComboBoxItem cairoImageWallpaperItem = new ComboBoxItem()
+            {
+                Name = "cairoImageWallpaper",
+                Content = Localization.DisplayString.sSettings_Desktop_BackgroundType_cairoImageWallpaper,
+                Tag = cairoImageBackgroundStackPanel
+            };
+            ComboBoxItem cairoVideoWallpaperItem = new ComboBoxItem()
+            {
+                Name = "cairoVideoWallpaper",
+                Content = Localization.DisplayString.sSettings_Desktop_BackgroundType_cairoVideoWallpaper,
+                Tag = cairoVideoBackgroundStackPanel
+            };
+            ComboBoxItem bingWallpaperItem = new ComboBoxItem()
+            {
+                Name = "bingWallpaper",
+                Content = Localization.DisplayString.sSettings_Desktop_BackgroundType_bingWallpaper,
+                Tag = bingImageBackgroundStackPanel
+            };
+
+            cboDesktopBackgroundType.Items.Add(windowsDefaultBackgroundItem);
+
+            if (Shell.IsCairoUserShell)
+            {
+                cboDesktopBackgroundType.Items.Add(cairoImageWallpaperItem);
```

Comment:
Instead of this can we simply hide the dropdown if not Startup.IsCairoUserShell?

Reply:
This was a kinda hack to ensure that if Explorer is shell that the only option available was the windows settings.